packages/babel-plugin-transform-es2015-parameters/src/default.js

```diff
@@ -1,137 +0,0 @@
-import getFunctionArity from "babel-helper-get-function-arity";
```

Comment:
Hi! It's quite alright about the overlap. I actually made that assumption on my own. After making the functionality work correctly, I thought I should learn about the codebase more and try to integrate it with proper tests. 

There were two ways of putting the code on the repo and showing it to you guys, either add this as a different plugin, something like a `generic-inline-parameters-in-body`and then `es2015-parameters` could use this. Or it could replace the `es2015-parameters`. I couldn't figure out which way to go so I just made the call of going with the latter. I haven't spent too much time on it and it could be easily re-purposed as a function which you mention. 

And I opened this PR just to get a code review on it (and to show the transformations its making like in the tests) since it's more proper than an astexplorer link :P (half the things don't work there btw because they're using an older version of babel and babylon). I could easily repurpose this into a method in babel-traverse/conversion.js.

So should I do that and open a new PR? Or should I let the reviews come to this PR on the functionality I mentioned and then make a new PR after the reviews? (I guess the former would be better, to avoid confusion. But please let me know)